WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -75,6 +93,21 @@ import WordPressShared
     // MARK: - Configuration
 
 
+    func setupRefreshControl() {
+        if refreshControl != nil {
```

Comment:
Ah no biggie at all, just following the idea that `guard` clarifies things by writing the condition you _do_ want rather than what you don't. This is so simple though, I probably should have left it alone 🙂